api/resources_portal/test/views/test_organization.py

```diff
@@ -49,46 +55,83 @@ def setUp(self):
         self.url = reverse("organization-detail", args=[self.organization.id])
 
     def test_get_request_returns_a_given_organization(self):
+        user = UserFactory()
+        self.client.force_authenticate(user=user)
         response = self.client.get(self.url)
         self.assertEqual(response.status_code, status.HTTP_200_OK)
 
-    def test_put_request_updates_a_organization(self):
+    def test_get_requires_account(self):
+        response = self.client.get(self.url)
+        self.assertEqual(response.status_code, 403)
+
+    def test_put_request_fails_by_stranger(self):
```

Comment:
The test name is a little unclear. Is this testing that a PUT request by the wrong person fails?